src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -476,9 +476,7 @@ m2mtest_bank(xclDeviceHandle handle, boost::property_tree::ptree& _ptTest, int b
   }
 
   //bandwidth
-  size_t total = bo_size;
-  total *= 1000000; // convert us to s
-  total /= (1024 * 1024); //convert to MB
+  size_t total = bo_size / (1024 * 1024); //convert to MB
```

Comment:
Could you change the name of these variables to indicate their contents.   For example:
`total` becomes `total_Mb`
`timer_stop` becomes `timer_duration_sec`